src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { createStore, compose, applyMiddleware } from 'redux';
+import reducer from './reducers/index';
+import App from './containers/App';
+
+const createStoreWithMiddleware = compose(applyMiddleware(thunk))(createStore);
+const store = createStoreWithMiddleware(reducer);
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
```

Comment:
please add an empty line as last line in every file (eslint)

Reply:
@ChristianSchmidt1981 
It has the empty line but it is not showing up.